ui/apps/platform/src/Containers/Violations/Details/Deployment/ContainerConfiguration.tsx

```diff
@@ -74,11 +75,15 @@ function ContainerConfiguration({ container }): ReactElement {
     );
 }
 
-function ContainerConfigurations({ deployment }): ReactElement {
+export type ContainerConfigurationsProps = {
+    deployment: Deployment | null;
+};
+
+function ContainerConfigurations({ deployment }: ContainerConfigurationsProps): ReactElement {
     return (
         <Card isFlat>
             <CardBody>
-                {deployment?.containers?.length > 0
+                {deployment && deployment.containers.length > 0
```

Comment:
You know I do not trust any API to do what they say they will do.

Because if they don't, and the UI crashes at runtime, guess who gets the blame? (hint: it isn't the API)

```suggestion
                {deployment && deployment?.containers?.length > 0
```